templates/content/node--oe-project--oe-w-content-banner.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Project content banner display.
+ */
+#}
+{{ pattern('content_banner', {
+  'background': 'gray',
+  'title': label,
+  'content': content.oe_teaser,
+  'image': image,
+  'attributes': create_attribute().addClass(['ps-0']),
+  'badges': content.oe_subject
+    |field_value|default([])
+    |filter(_item => _item|drupal_escape|striptags|trim is not empty)
+    |map(_item => {
+      'label': _item,
+      'rounded_pill': true,
+    }),
```

Comment:
I don't like it, but I also don't like the for loops with array merge to add new items.

So:
- `|field_value` is like `element_children()`, but unfortunately it returns NULL sometimes. Both `|filter` and `|map` below require an array as parameter, so we need `|default([])`.
- The `|filter()` cleans out field items that are empty when rendered. Perhaps we can skip this step.
- `|map` does the actual array structure conversion.

I will test if empty leftover field items are a thing. If not, I will drop `|filter()`.

Reply:
Actually `|drupal_escape` triggers rendering.
On the other hand, `|field_value` does not.
But ok let's keep. If we have to change this here, there are plenty of other places where we do something that will have the same effect.

Reply:
"Hey this can lead to a bug, but we did it a bit everywhere so let's ignore it". Sounds like a flawed approach to me :)
But anyway I'll make a follow-up to replace field_value with our own filter as we discussed.

Reply:
I did some digging.
Indeed some formatters add cache info directly to the `$elements` array, which is lost if we pick the children.
This will be lost with `|field_value`, but also when using a `for` loop to pick field items.

Let's indeed fix this with a new issue.